### ISSUE TITLE
Raygui.List_view_ex documented incorrectly

### ISSUE THREAD

**Issue by kevinclancy** [tjammer/raylib-ocaml#56]
When I create a list view using `Raygui.list_view_ex`, I'm unable to get its selection to change. Focus (mouse hover) for list items seems to be working. But when I click on a list item, I expect the third result returned (`selected_index` below) to be the index of the selected item. However, even when I click on item 1, the value `0` is returned. The `Raygui.list_view` function seems to have the same problem.

```
    let scroll_index, focused_index, selected_index =
      Raygui.list_view_ex
        boundary
        ["blib" ; "blab"]
        !(region_editor.focused_region_index)
        !(region_editor.region_list_scroll_index)
        !(region_editor.selected_region_index)
    in
    region_editor.region_list_scroll_index := scroll_index;
    region_editor.focused_region_index := focused_index;
    region_editor.selected_region_index := selected_index;
```

**Comment by kevinclancy** [comment]
By looking at the gui test, I figured out what's wrong. The selected index is actually the first result, which I'm storing in scroll_index. 

This isn't documented correctly. Looking at the doc comment, in the parameters the selected item index is called `active`. In the results, the selected item index is called "selected list item index". I find the latter to be far more helpful. I propose changing the name of the `active` parameter to `selected_item_index` and changing the name of the third result to `scroll_index`.

**Comment by tjammer** [comment]
Thanks for reporting this, and finding out what's wrong. `active` and `selected list item index` are really the same thing, I must have thought the `index` in the implementation refers to the selected index (i.e. `active`) instead of `scroll_index`.

I want to keep the parameter name as `active`. Not because I think it's a good name (it's not), but because that's the name raygui uses.

I fixed the doc comments in 3063dda4587be5538e8f2d54621de1ee3847b540.
